src/core/geometry/qgsgeometryutils.h

```diff
@@ -291,6 +291,31 @@ class CORE_EXPORT QgsGeometryUtils
      */
     static QgsPointV2 midpoint( const QgsPointV2& pt1, const QgsPointV2& pt2 );
 
+    /** Return the slope of a line defined by points pt1 and pt2.
+     * @param pt1 first point.
+     * @param pt2 second point.
+     * @return The slope of this linear entity, or infinity if vertical
```

Comment:
Probably worth mentioning that the slope returned is a gradient, and not an angle